demos/twitter-race/README.md

```diff
@@ -0,0 +1,49 @@
+# Twitter Race
+
+This demo shows reading from a live stream of tweets using the [twitter adapter](https://github.com/juttle/juttle-twitter-adapter).
+The user inputs 2 search terms, juttle parses the incoming tweets to find matches, and displays match count as live-updating tiles and a timechart,
+to visually compare popularity of the requested terms.
+
+## Setup
+
+We will need outrigger to render visualizations in the browser, and the twitter adapter to enable reading. 
+
+```
+npm install -g juttle
+npm install -g outrigger
```

Comment:
Once https://github.com/juttle/outrigger/issues/48 is fixed we won't need to explicitly install juttle or the adapter. I think it makes sense to future proof these instructions and just tell people to install outrigger and then configure the adapter.